test/models/comment_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+   test "comment_on_post" do 
```

Comment:
We actually don't need any model tests for comments.  The `comment_on_post` isn't actually doing a lot.  Instead we should have a `comments_controller_test`.